examples/webserver/webserver.py

```diff
@@ -0,0 +1,63 @@
+import os
+import asyncio
+import fs_async
+from microdot import Microdot, send_file
+from config import base_dir
+
+extra_content_type_map = {
+    'pdf': 'application/pdf',
+    'svg': 'image/svg+xml',
+}
+
+class FileStream:
```

Comment:
```suggestion
# We use a custom FileStream so we are not restricted to the buffer size that microdot chooses.
class FileStream:
```

Reply:
Not sure if this is 100% correct but something along the lines of this would be good.